corehq/apps/app_manager/models.py

```diff
@@ -4125,7 +4125,7 @@ def validate_for_build(self):
                 'type': 'report config ref invalid',
                 'module': self.get_module_info()
             })
-        if not self.reports:
+        elif not self.reports:
```

Comment:
not sure what this get's us. If there are no reports then the previous if won't ever execute.